recipes/gpgme/recipe/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: gpgme
+  version: 1.13.1
+
+source:
+  url: https://gnupg.org/ftp/gcrypt/gpgme/gpgme-1.13.1.tar.bz2
+  sha256sum: c4e30b227682374c23cddc7fdb9324a99694d907e79242a25a4deeedb393be46
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - {{ compiler('c') }}
```

Comment:
Move this to build then you will get rid of the error mentioned in the PR comment. Also add `make` to build dependencies.

Reply:
right, will do!